static/js/message_events.js

```diff
@@ -395,6 +402,7 @@ export function update_messages(events) {
                 // If an event changed both content and topic, we'll generate
                 // two client-side events, which is probably good for display.
                 const edit_history_entry = {
+                    // BUG: event.edited_by field does not exist
                     edited_by: event.edited_by,
```

Comment:
Maybe let's fix the `edited_by` bug in a quick prep commit? It seems orthogonal and also kinda embarrassing :)